modules/core/04-channel/v2/keeper/grpc_query_test.go

```diff
@@ -410,6 +410,119 @@ func (suite *KeeperTestSuite) TestQueryPacketAcknowledgement() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestQueryPacketAcknowledgements() {
+	var (
+		req                 *types.QueryPacketAcknowledgementsRequest
+		expAcknowledgements = []*types.PacketState{}
+	)
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+		expError error
+	}{
+		{
+			"success",
```

Comment:
nit: maybe the name could be a bit more descriptive since it tests some quite specific behavior? Something like this?

```suggestion
			"success: with PacketCommitmentSequences",
```